native/src/components/Header.tsx

```diff
@@ -82,6 +84,7 @@ const Header = ({
   // Save route/canGoBack to state to prevent it from changing during navigating which would lead to flickering of the title and back button
   const [previousRoute] = useState(navigation.getState().routes[navigation.getState().routes.length - 2])
   const [canGoBack] = useState(navigation.canGoBack())
+  const { setVisible, content } = useTtsPlayer()
```

Comment:
```suggestion
  const { setVisible: setTtsPlayerVisible , content } = useTtsPlayer()
```
To have a better idea whats happens if you use it below